### PR TITLE
Add CT shaded relief

### PR DESCRIPTION
**Pull request by Mashin6**
Adding shaded relief tiles from CT ECO.
image

### CODE REVIEW

**Comment by cicku** [comment]
Please include year in the name.

File: sources/north-america/us/ct/CT_ECO_Shaded_relief.geojson [hunk 1]
```diff
@@ -0,0 +1,468 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "CT_ECO_Shaded_relief",
```
Comment (cicku) [inline sources/north-america/us/ct/CT_ECO_Shaded_relief.geojson]: Year

Comment (Mashin6) [reply sources/north-america/us/ct/CT_ECO_Shaded_relief.geojson]: Does the file name have to be changed as well?

Comment (cicku) [reply sources/north-america/us/ct/CT_ECO_Shaded_relief.geojson]: I'd recommend doing that, both id and filename

**Comment by cicku** [comment]
Merged, thanks!

**Pull request merged**